docs/devel.md

```diff
@@ -146,6 +146,57 @@ The service log will show:
 2020-06-08 15:12:39,478 127.0.0.1 - - [08/Jun/2020 15:12:39] "GET /repos/app-sre/sretoolbox HTTP/1.1" 200 -
 ```
 
+## Run github-mirror in Redis backed mode (optional)
+
+When running in Redis mode, github-mirror will cache all requests in a Redis server instead of locally in-memory. This enables multiple instances of github-mirror to access and maintain a single shared cache.
+
+### A. Configuring Redis
+First make sure to download and install [Redis](https://redis.io/download).
+
+Since the default Redis configuration does not require authentication, you will also need to create a Redis configuration file.
+
+Create a simple configuration file `/etc/redis/sample_config.conf` with a Redis authentication password:
+
+```
+requirepass <mysecret>
+```
+
+From a different terminal run the Redis server:
+
+```
+user@localhost:~$ redis-server /etc/redis/sample_config.conf
+```
+
+Use the Redis client `redis-cli` to test your connection to the Redis server.
+
+By default, `redis-cli` will connect to the server at `127.0.0.1` port `6379`:
+```
+user@localhost:~$ redis-cli -a <mysecret>
+127.0.0.1:6379> ping
+PONG
+```
```

Comment:
I'm not sure we should approach redis config here, even if in a very simplistic way... I'd rather just have a docker command line, like:
```
docker run --rm -it -p 6379:6379 redis
```
That's is enough for testing.

Reply:
I don't think you can set the authentication that way. you'd still have to supply a config file.

and you'd still want to use the redis client to test the connection...

Reply:
I see... there's more to it indeed.